web/src/app/alerts/CreateAlertDialog/StepContent/Step2.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Typography, Grid, Divider } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import { ServiceChip } from '../../../util/Chips'
+
+const useStyles = makeStyles(theme => ({
+  divider: {
+    backgroundColor: '#bdbdbd',
+  },
+  itemTitle: {
+    paddingBottom: 0,
+  },
+  nudgeRight: {
+    marginLeft: theme.spacing(1),
+  },
+}))
+
+export default props => {
```

Comment:
Same as above, let's name this component more descriptively and give the function the same name